tests/integration/security/reachability/context.go

```diff
@@ -111,6 +111,7 @@ func (rc *Context) Run(testCases []TestCase) {
 	}
 
 	for _, c := range testCases {
+		c := c
```

Comment:
Can you add a comment for this assignment so we don't get into the same trouble next time.

Reply:
done